src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4994,11 +4996,11 @@ namespace ZoneEquipmentManager {
                         retNodeData.MassFlowRateMaxAvail = zoneNodeData.MassFlowRateMaxAvail;
                     }
 
-                    airLoopReturnFrac = DataAirLoop::AirLoopFlow(airLoop).DesReturnFrac;
-                    if (DataAirSystems::PrimaryAirSystem(airLoop).OASysExists) {
-                        // Set return flow as fraction of matching inlet node flow if there is an OA system
+                    airLoopReturnFrac = thisAirLoopFlow.DesReturnFrac;
+                    if (DataAirSystems::PrimaryAirSystem(airLoop).OASysExists && (thisAirLoopFlow.OAFlow > 0.0)) {
+                        // Set return flow as fraction of matching inlet node flow if there is an OA system and OA flow > 0.0
                         returnNodeMassFlow = airLoopReturnFrac * inletMassFlow;
-                        thisZoneEquip.TotAvailAirLoopOA = DataAirLoop::AirLoopFlow(airLoop).MaxOutAir;
+                        thisZoneEquip.TotAvailAirLoopOA += thisAirLoopFlow.OAFlow;
```

Comment:
So if there are 2 zone equipment does each thisZoneEquip.TotAvailAirLoopOA = air loop OA? Wouldn't that somehow double count OA?

Reply:
How is "int airLoop = thisZoneEquip.ReturnNodeAirLoopNum(returnNum);" associated with a specific zone equipment?

Reply:
Say there are two airloops serving a zone and both airloops have OA available, and there is an exhuast fan. The intent here is to reduce the return flows proportionally to the amount of OA each system is bringing in. Not sure as I'm writing this if TotAVailAirLoopOA should even be connected to a zone equip, it may just need to be a local value since this function goes zone-by-zone.  Need to review that.